ext/vscode/package.nls.json

```diff
@@ -90,5 +92,23 @@
     "azure-dev.surveys.surveyOne.prompt": "Can you please take 2 minutes to tell us how the Azure Developer CLI is working for you?",
     "azure-dev.surveys.surveyOne.button.take": "Take survey",
     "azure-dev.surveys.surveyOne.button.never": "Don't ask again",
-    "azure-dev.surveys.surveyOne.button.later": "Remind me later"
+    "azure-dev.surveys.surveyOne.button.later": "Remind me later",
+
+    "azure-dev.walkthroughs.start.title": "Deploy your App with the Azure Developer CLI",
+    "azure-dev.walkthroughs.start.description": "Your quick guide to set up, provision, and deploy a new app with the AZD CLI.",
+
+    "azure-dev.walkthroughs.start.steps.install.title": "Install Azure Developer CLI",
+    "azure-dev.walkthroughs.start.steps.install.description": "Install the Azure Developer CLI (**azd**) to get started with Azure development.\n[Install](command:azure-dev.commands.cli.install)",
+
+    "azure-dev.walkthroughs.start.steps.login.title": "Sign In to your Azure Account",
+    "azure-dev.walkthroughs.start.steps.login.description": "Sign in or create an Azure account to successfully run **azd** commands.\n[Sign In](command:azure-dev.commands.cli.login)\nTip: Clicking the **Sign In** button may open a new browser window with a prompt to input your Azure credentials.",
+
+    "azure-dev.walkthroughs.start.steps.scaffold.title": "Scaffold the Code",
+    "azure-dev.walkthroughs.start.steps.scaffold.description": "View the [Awesome azd template gallery](https://azure.github.io/awesome-azd/) to find a project with the infrastructure that's right for you, or select a template directly from the Command Palette (``azd init``).\n[Select a template and initialize your project](command:azure-dev.commands.cli.init)\nTip: To use a template from the Awesome azd site, copy its GitHub repo URL and enter it when prompted during ``azd init``.",
```

Comment:
Can we reword "View the [Awesome azd template gallery](https://azure.github.io/awesome-azd/) to find a project with the infrastructure that's right for you"

To something like:

"View the [Awesome azd template gallery](https://azure.github.io/awesome-azd/) to find a template with the infrastructure that's right for your app" 

I want to make sure that the value prop of the the templates (IaC + editor configs etc) isn't lost here. Trying to think about how to frame this in the most compelling way.

@gkulin thoughts?